File: FizzBuzz.go
package main

import (
	"fmt" 
	"strconv"
)

func fizzbuzz(num int) string {
	fb := ""
	
	if num % 3 == 0 {
		fb += "Fizz"
	}
	if num % 5 == 0 {
		fb += "Buzz"
	}
	if fb == "" {
		fb = strconv.Itoa(num)
	}

	return fb
}

func main() {
	for i := 1; i <= 100; i++ {
		fmt.Println(fizzbuzz(i))
	}
}

File: FizzBuzz.py
def fizzbuzz(num):
    return "Fizz" * (num % 3 == 0) + "Buzz" * (num % 5 == 0) or str(num)

if __name__ == '__main__':
    for i in range(1, 101):
        print(fizzbuzz(i))